OrPinhas_and_AvivShabtay_Project/OrPinhas_and_AvivShabtay/OrPinhas_and_AvivShabtay/Person.h

```diff
@@ -12,15 +12,15 @@ class Person
 public:
 	Person(int age, char* strName)
```

Comment:
Convert `strName` to: `name`
It's simpler and understandable from the type (`char*`)